layouts/partials/cncf.html

```diff
@@ -0,0 +1,9 @@
+<div class="main-section">
+	<center>
+    	<p style="font-size: 20px">{{ T "main_cncf_project" | safeHTML }}
+    	<picture>
+	  		<source srcset="/images/cncf-white.png" media="(prefers-color-scheme: dark)">
+	  		<img src="/images/cncf-color.svg" class="cncf-img">
```

Comment:
We should use SVG for both light and dark mode. See https://github.com/cncf/artwork/blob/main/examples/other.md#cncf-logos